### ISSUE TITLE
Load WebView on initial render

### ISSUE THREAD

**Issue by sjransom** [react-native-webview/react-native-webview#2431]
**Bug description:**
I'm trying to preload my WebView in a tab using React Navigations Bottom Tabs Navigator and the prop [lazy](https://reactnavigation.org/docs/bottom-tab-navigator/#lazy) set to `false` so that the screen is loaded on initial render.

Unfortunately while the screen renders the actual WebView component does not until the tab is active, this appears to only affect iOS.

**To Reproduce:**
Set `lazy: false` to screenOptions prop of Tab.navigator or options prop of Tab.Screen and render WebView as a component on the screen.

**Expected behavior:**
WebView component should be loaded on initial render.

**Environment:**
 - OS: iOS
 - OS version: 15.4
 - react-native version: 0.67.4
 - react-native-webview version: 11.17.2

If there are other known working solutions to preload WebView please let me know.

**Comment by walterholohan** [comment]
We are experiencing the same issue on our React Native App which is causing a terrible user experience. When a user switches to a tab using the bottom tab bar there have to wait nearly 5 seconds for the Webview to load. If the above issue was fixed it would be amazing as the Webview would already be loaded when the user lands on that tab.

**Comment by yunjeongloper** [comment]
+1

**Comment by jinee525** [comment]
:+1:

**Comment by sunju0123** [comment]
I am also having the same problem. It's very bad for the user experience.

**Comment by dom-runna** [comment]
Also the same would be amazing if the team would be able to fix please!

**Comment by gbhrdt** [comment]
I was able to achieve preloading with the here mentioned patch (using NPM patch-package): https://github.com/react-native-webview/react-native-webview/pull/1852#issuecomment-1120470181

**Comment by walterholohan** [comment]
@gbhrdt do you know if the patch has been merged into master yet?

**Comment by finalight** [comment]
is this still in progress?

**Comment by walterholohan** [comment]
@finalight I believe @gbhrdt created a patch for it https://github.com/react-native-webview/react-native-webview/pull/1852#issuecomment-1120470181 however did not create a PR. @gbhrdt does your solution work? If so can you raise a PR?

**Comment by walterholohan** [comment]
@finalight a PR was just raised with the fix, we are currently using this fix in our production app and it works great

https://github.com/react-native-webview/react-native-webview/pull/2626

**Comment by ivona314** [comment]
Eagerly waiting for this to be released

**Comment by gronxb** [comment]
The same thing happens in the  >= 13.2.3 version because it was revert due to that PR.
https://github.com/react-native-webview/react-native-webview/pull/3041

I think this issue should be reopened.

**Comment by Santhosh-Umapathi** [comment]
`"react-native-webview": "^13.10.0"`

Still the same issue, has to be reopened

**Comment by LoadJulz** [comment]
> The same thing happens in the >= 13.2.3 version because it was revert due to that PR. #3041
> 
> I think this issue should be reopened.

Did you manage to get it work to load the webview tab eagerly in 13.2.2? I have tried it but it still only loads when the tab is open.

**Comment by jamonholmgren** [comment]
Fix incoming.

**Comment by badalsaibo** [comment]
Should be reopened since the fixed commit was reverted due to a regression.
https://github.com/react-native-webview/react-native-webview/pull/3615

**Comment by yurimutti** [comment]
Was fixed ?

**Comment by LoadJulz** [comment]
Seems fixed. Just checked 13.15.0
